app/models/adopter_searcher.rb

```diff
@@ -37,7 +37,7 @@ def search
 
     if @params[:search]
       if email_search?
-        @adopters = @adopters.where('adopters.email ILIKE ?', "%#{@params[:search].strip}%")
+        @adopters = @adopters.where('adopters.email ILIKE ? OR adopters.secondary_email ILIKE ?', "%#{@params[:search].strip}%", "%#{@params[:search].strip}%")
```

Comment:
strip once then use that value

```suggestion
        clean_email = @params[:search].strip
        @adopters = @adopters.where('adopters.email ILIKE ? OR adopters.secondary_email ILIKE ?', clean_email, clean_email)
```